docs/about.md

```diff
@@ -0,0 +1,49 @@
+## Mailing list
+
+OpenBLAS users: <https://groups.google.com/forum/#!forum/openblas-users>
+
+OpenBLAS developers: <https://groups.google.com/forum/#!forum/openblas-dev>
+
+## Donations
+
+Thank you for the support.
+
+You can read OpenBLAS statement of receipts and disbursement and cash balance on [google doc](https://docs.google.com/spreadsheet/ccc?key=0AghkTjXe2lDndE1UZml0dGpaUzJmZGhvenBZd1F2R1E&usp=sharing).
+
+### Fundraiser
+
+* [2013.8] [Testbed for OpenBLAS project](https://www.bountysource.com/fundraisers/443-testbed-for-openblas-project)
+
+Here is [Backer list](https://github.com/xianyi/OpenBLAS/blob/develop/BACKERS.md).
+
+### For Personal Donation
+
+You can create a bounty for an issue on [bountysource.com](https://www.bountysource.com/trackers/69691-openblas). 
```

Comment:
Bountysource is dead and kept money for completed work rather than paying it out at the end. They got taken over and then it went downhill fast. So let's delete this. And the link higher up as well.